pkgs/servers/headscale/default.nix

```
@@ -8,13 +8,13 @@
 }:
 buildGoModule rec {
   pname = "headscale";
-  version = "0.25.0";
+  version = "0.25.1";
 
   src = fetchFromGitHub {
     owner = "juanfont";
     repo = "headscale";
     rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
while we're here and since you're not a bot